donut-kata/src/main/kotlin/bnymellon/codekatas/donutkatakotlin/DonutType.kt

```diff
@@ -1,18 +1,6 @@
-/*
- * Copyright 2020 The Bank of New York Mellon.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
+// Copyright 2021 BNYMellon. All rights reserved.
```

Comment:
Should we change the kotlin files to have copyright notices similar to Java files?